src/module.ts

```diff
@@ -0,0 +1,157 @@
+import { PanelConfig } from './panel-config';
+import { PanelCtrl, loadPluginCss } from 'grafana/app/plugins/sdk';
+import * as _ from 'lodash';
+import * as go from '../node_modules/gojs/release/go';
+import * as axios from '../node_modules/axios/dist/axios';
+
+class Ctrl extends PanelCtrl {
+  static templateUrl = "partials/template.html";
+
+  private panelDefaults = {
+    host: "localhost",
+    port: "7890",
+    application: "Servers,1.0.0",
+    user: "admin",
+    password: "admin"
+  };
+  private myFullDiagram: any = {
+    div: undefined,
+    nodeTemplate: undefined,
+    groupTemplate: undefined,
+    linkTemplate: undefined,
+    model: undefined
+  };
+  private setupDiagramTimer: number | undefined = undefined;
+  private _panelConfig: PanelConfig;
+
+  constructor($scope: any, $injector) {
+    super($scope, $injector);
+    this._panelConfig = new PanelConfig(this.panel);
+    _.defaultsDeep(this.panel, this.panelDefaults);
+
+    this.events.on('init-edit-mode', this._onInitEditMode.bind(this));
```

Comment:
No dispose event ?

Reply:
No dispose event is used in similar projects https://github.com/CorpGlory/grafana-progress-list/blob/master/src/module.ts